src/conf_mode/protocols_isis.py

```diff
@@ -203,6 +203,37 @@ def verify(isis):
         if list(set(global_range) & set(local_range)):
             raise ConfigError(f'Segment-Routing Global Block ({g_low_label_value}/{g_high_label_value}) '\
                               f'conflicts with Local Block ({l_low_label_value}/{l_high_label_value})!')
+        
+    # Check for a blank or invalid value per prefix
+    if dict_search('segment_routing.prefix', isis):
+        for prefix in isis['segment_routing']['prefix']:
```

Comment:
Please use: `for prefix, prefix_config in isis['segment_routing']['prefix'].items():` this unbloads the code below as you can replace `isis['segment_routing']['prefix'][prefix]` by `prefix_config`.

Reply:
Sure, ok cool. I will use that instead.